### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by Noxcivis** [Peppermint-Lab/peppermint#371] [Ideas]
Any update on Dark Mode ?  I'm very much looking forward to using Peppermint but am eagerly awaiting the Dark Mode option that was hinted at.

**Comment by potts99** [comment]
soon 👍 i have quite a lot of it done, but i might release it with full theme support rather than light/dark only.

**Comment by potts99** [comment]
@Noxcivis this is now implemented!!
